src/macros.jl

```diff
@@ -358,6 +358,47 @@ end
 
 # two-argument build_constraint is used for one-sided constraints.
 # Right-hand side is zero.
+
+"""
+    sense_to_set(_error::Function, ::Val{sense_symbol})
+
+Converts a sense symbol to a set `set` such that
+`@constraint(model, func sense_symbol 0) is equivalent to
+`@constraint(model, func in set)` for any `func::AbstractJuMPScalar`.
+
+## Example
+
+Once a custom set is defined you can directly create a JuMP constraint with it:
+```jldoctest sense_to_set; setup = :(using JuMP)
```

Comment:
Nit: Why is `using JuMP` needed for the doctest while `MOIU` doesn't need to be defined?

Reply:
`MOIU` is exported by JuMP